### ISSUE TITLE
Sort the views alphabetically

### ISSUE THREAD

**Issue by nicolas-goudry** [bluskript/nix-inspect#11]
It would be great if views were sorted alphabetically. Searching visually for an attribute would be far much easier (I know there is the search feature, but still).

**Comment by nicolas-goudry** [comment]
To further advocate for this feature, since the search feature doesn’t allow to navigate between results, it would ease the process.

As an example, my `config.environment` list is the following (shortened):
```
sessionVariables
systemPackages
etc
enableDebugInfo
...
gnome
lxqt
profileRelativeSessionVariables
profileRelativeEnvVars
profiles
...
gnome3
blcr
checkConfigurationOptions
```

In this case, if I search for `gno` the `gnome` entry will be highlighted. However, as a vim user I would expect to <kbd>Enter</kbd> and get the <kbd>n</kbd> / <kbd>N</kbd> key to move between results. I understand that when I hit <kbd>Enter</kbd> it will enter the current match, so having an alphabetical sort would allow me to just hit <kbd>j</kbd> to highlight `gnome3` and be able to enter it quickly.

**Comment by bluskript** [comment]
I agree that sorting the attrset alphabetically might make more sense. FYI, the "go-to path" functionality supports tab completion and might help with moving through results in the meantime. Search doesn't do this for now

**Comment by xdbr** [comment]
+1
